docs/create_your_first_nn.md

```diff
@@ -165,6 +165,123 @@ Total params: 266610
+    it.fit(
+        dataset = train,
+        epochs = 10,
+        batchSize = 100
+    )
+
+    val accuracy = it.evaluate(dataset = test, batchSize = 100).metrics[Metrics.ACCURACY]
+
+    println("Accuracy: $accuracy")
+    it.save(File("model/my_model"), writingMode = WritingMode.OVERRIDE)
+}
+```
+
+Here are some important parameters that we need to pass to the `fit()` method:
+* `epochs` - Number of iterations over the data you want the training process to perform. Epoch = iteration.
+* `batchSize` - How many examples will be used for updating the model's parameters (aka weights and biases) at a time.
+
+After the model has been trained, it's important to evaluate its performance on the test set, so that we can check how it generalizes to the new data.
+
+```kotlin
+val accuracy = it.evaluate(dataset = test,
+    batchSize = 100).metrics[Metrics.ACCURACY]
+
+println("Accuracy: $accuracy")
+```
+
+```
+Accuracy: 0.8821001648902893
+```
+
+---
+**NOTE**
+
+The results are nondeterministic, and you may have a slightly different Accuracy value.
+
+--- 
+
+When we are happy with the model's evaluation metric, we can save the model for future use in the production environment.
+
+```kotlin
+it.save(File("model/my_model"), writingMode = WritingMode.OVERRIDE)
+```
+
+And just like that, we have trained, evaluated, and saved a deep learning model that we can now use to generate predictions (aka inference).
+
+## Loading trained model for inference
+
+In this section let's look at how to load and use a trained model to generate predictions on new, previously unseen data.
+
+For illustration purposes, and to simplify this tutorial,
+we'll use the test data subset to generate a prediction example that the model has not been trained on.
+
+The example images in the test data have the same size and format as the ones the model has been trained on,
+so we do not need to do any additional preprocessing.
+However, if you are going to train the model on your own data,
+make sure to use the same image preprocessing before using an image for inference as you did while training your model.
+Every model expects to get exactly the same input as it was trained on.
+
+To load the model simply use the path to it, tell it how incoming images should be reshaped (if needed), and call the
+`predict` method on them.
+
+```kotlin
+fun main() {
+    val (train, test) = fashionMnist()
+
+    TensorFlowInferenceModel.load(File("model/my_model")).use {
+        it.reshape(28, 28, 1)
+        val prediction = it.predict(test.getX(0))
+        val actualLabel = test.getY(0)
+
+        println("Predicted label is: $prediction. This corresponds to class ${stringLabels[prediction]}.")
+        println("Actual label is: $actualLabel.")
+    }
+}
+```
+
+```
+Predicted label is: 9. This corresponds to class Ankle boots.
+Actual label is: 9.0.
+```
+
+Congratulations! You have learned how to create, train, and use your first neural network model!
```

Comment:
Congratulations! You have learned to create, train, and use your first neural network model!